webdriver/tests/bidi/script/evaluate/invalid.py

```diff
@@ -0,0 +1,78 @@
+import pytest
+import webdriver.bidi.error as error
+from ... import recursive_compare
+
+pytestmark = pytest.mark.asyncio
+
+
+@pytest.mark.parametrize("context", [None, False, 42, {}, []])
+async def test_params_context_invalid_type(bidi_session, context):
+    with pytest.raises(error.InvalidArgumentException):
+        await bidi_session.script.evaluate(
+            expression="1 + 2",
+            target=bidi_session.script.ContextTarget(context))
+
+
+async def test_params_context_unknown(bidi_session):
+    with pytest.raises(error.NoSuchFrameException):
+        await bidi_session.script.evaluate(
+            expression="1 + 2",
+            target=bidi_session.script.ContextTarget("_UNKNOWN_"))
+
+
+@pytest.mark.parametrize("realm", [None, False, 42, {}, []])
+async def test_params_realm_invalid_type(bidi_session, realm):
+    with pytest.raises(error.InvalidArgumentException):
+        await bidi_session.script.evaluate(
+            expression="1 + 2",
+            target=bidi_session.script.RealmTarget(realm))
+
+
+async def test_params_realm_unknown(bidi_session):
```

Comment:
```suggestion
async def test_params_realm_invalid_value(bidi_session):
```

Reply:
`test_params_realm_invalid_type` should raise `InvalidArgumentException`, while `test_params_realm_unknown` should raise `NoSuchFrameException`. So I just split them into 2 different tests.